goroutine/gid_test.go

```diff
@@ -19,8 +19,12 @@ func TestID(t *testing.T) {
 	if got != want {
 		t.Fatalf("unexpected id for main goroutine: got:%d want:%d", got, want)
 	}
+	n := 1000000 // for regular go
+	if IsTinyGo {
+		n = 1000 // for tinygo
+	}
```

Comment:
This comment could be improved.

It explains what you are doing, but not why you are using a lower value for tinygo

Reply:
tinygo OOMs but yes let me put that in